src/systemd/sd-dhcp-static-lease.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include "_sd-common.h"
+
+extern const struct hash_ops dhcp_static_leases_hash_ops;
```

Comment:
should not be in public headers. Again, public headers should only have sd_xyz symbols.